code/modules/mining/equipment/survival_pod.dm

```diff
@@ -112,6 +112,7 @@
 	icon = 'icons/obj/doors/airlocks/survival/survival.dmi'
 	overlays_file = 'icons/obj/doors/airlocks/survival/survival_overlays.dmi'
 	assemblytype = /obj/structure/door_assembly/door_assembly_pod
+	CanAtmosPass = ATMOS_PASS_NO
```

Comment:
fuck invisible atmos barriers,

all my homies hate invisible atmos barriers.

Reply:
like how is this a better alternative to tiny fnas?

Reply:
i added a holobarrier to the door, it makes more sense than a tiny fan that blocks all air by "a gust of wind"